frontend/src/app/libs/user-profile-page/service/user-service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { IUser } from '../model/user';
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { catchError } from 'rxjs/operators';
+import { Observable, throwError } from 'rxjs';
+
+const URL_PREFIX: string = '/api';
+const ERROR_STATUS: number = 504;
+
+@Injectable({
+	providedIn: 'root',
+})
+export class UserService {
```

Comment:
Please merge latest master into your branch, a lot of changes in your commits.

Reply:
done